### PR TITLE
show version information

### PR DESCRIPTION
**Pull request by Its-Just-Nans**
may fix https://github.com/pdfarranger/pdfarranger/issues/909

```
touch tests/__init__.py
python3 -m unittest -v -f tests.test.TestBatch9
```

### CODE REVIEW

**Comment by kbengs** [comment]
This is how it looks on my machine:
image
Maybe it could be prettier.  :thinking:  `platform.freedesktop_os_release()` will give `PRETTY_NAME': 'KDE neon 6.2'` What do you think about that? (instead of platform.version())

TESTING.md changes could be in a separate commit. And as dreua commented before "Also the perfect commit message starts with a capital letter" "Show OS version information" :smiley:

**Comment by kbengs** [comment]
I'd vote for having only one line in about dialog:
Windows: "Running on" platform.platform() (Windows 11 is actually reported as Windows 10, don't know if this can be fixed)
Linux: "Running on" platform.freedesktop_os_release()["PRETTY_NAME"] (in a try except OSError, which return a empty string in case of exception)
Darwin: Have not tested

File: pdfarranger/pdfarranger.py [hunk 1]
```diff
@@ -702,6 +702,34 @@
...
+        """get the platform"""
+        os_type = platform.system()
+        try:
+            if os_type == 'Linux':
+                return platform.freedesktop_os_release()["PRETTY_NAME"]
+            elif os_type == 'Windows':
+                return platform.platform()
+            elif os_type == 'Darwin':
+                return platform.platform()
+        except Exception:
```
[the remark this answers is not in the document]
Comment (kbengs) [reply pdfarranger/pdfarranger.py:729]: Looks good to me. Add a comment to the except then codacy will be happy too.

[the remark this answers is not in the document]
Comment (kbengs) [reply pdfarranger/pdfarranger.py:730]: I think you need to have the comment just before the "pass" line. And a space between # and Ignore to follow the style used in the project.

**Comment by kbengs** [comment]
Could you clean the commit history so you have only two commits again. (force push)

**Comment by Its-Just-Nans** [comment]
You can just squash no ?

image

**Comment by kbengs** [comment]
@Its-Just-Nans Thank you!

**Pull request merged**